appsso-starter-java/build.gradle

```diff
@@ -19,13 +21,18 @@ dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-thymeleaf'
 
 	implementation 'org.springframework.boot:spring-boot-starter-oauth2-client'
-	implementation 'org.thymeleaf.extras:thymeleaf-extras-springsecurity5'
+	implementation 'org.thymeleaf.extras:thymeleaf-extras-springsecurity6'
 
 	implementation "org.webjars:webjars-locator-core:0.50"
 	implementation "org.webjars:bootstrap:5.1.3"
 	testImplementation 'org.springframework.boot:spring-boot-starter-test'
 }
 
+tasks.named("bootBuildImage") {
+	environment["BP_JVM_VERSION"] = "17"
+	buildpacks = ["urn:cnb:builder:paketo-buildpacks/java"]
```

Comment:
I'm curious why you need to specify this

Reply:
Removed.